dev/bots/docs.sh

```diff
@@ -125,9 +129,12 @@ DART="$DART_BIN/dart"
 PUB="$DART_BIN/pub"
 export PATH="$FLUTTER_BIN:$DART_BIN:$PATH"
 
-# Make sure dart is installed by invoking flutter to download it.
-"$FLUTTER" --version
-FLUTTER_VERSION=$(cat "$FLUTTER_ROOT/version")
+# Make sure dart is installed by invoking Flutter to download it.
+# This also creates the 'version' file.
+FLUTTER_VERSION=$("$FLUTTER" --version --machine)
+export FLUTTER_VERSION
+echo "$FLUTTER_VERSION"
```

Comment:
left-over debug print?

Reply:
No, it replaces the printing of the version that used to occur. I just wanted to a) log the version so that we have it in the CI logs, and b) not run `flutter version` more than once.

Reply:
Actually, I've realized that we print the version later on (from dartdoc.dart), so we don't need it here.